tensorboard/plugins/graph/tf_graph_common/tf-graph-icon.ts

```diff
@@ -74,7 +74,7 @@ namespace tf.graph.icon {
       },
     },
 
-    getSvgDefinableElement(): HTMLElement {
+    getSvgDefineableElement(): HTMLElement {
```

Comment:
Can we fix the call site instead? *Definable* is the correct spelling.

Reply:
:astonished:  Definable....? wow